tests/publiccloud/patch_and_reboot.pm

```diff
@@ -38,7 +38,14 @@ sub run {
     record_info('UNAME', $args->{my_instance}->ssh_script_output(cmd => 'uname -a'));
     $args->{my_instance}->ssh_assert_script_run(cmd => 'rpm -qa > /tmp/rpm-qa.txt');
     $args->{my_instance}->upload_log('/tmp/rpm-qa.txt');
+    $args->{my_instance}->cleanup_cloudinit() if (is_cloudinit_supported);
```

Comment:
but `is_cloudinit_supported` does not check if `PUBLIC_CLOUD_CLOUD_INIT` is actually set so we might get unwanted triggers of this code

Reply:
resolved